nexus-ingest/nexus-sink/src/test/groovy/org/nasa/jpl/nexus/ingest/nexussink/NexusSinkIntegrationTest.groovy

```diff
@@ -119,6 +138,89 @@ public class NexusSinkIntegrationTest {
+
+        def streamName = "testNexusSink"
+
+        NexusContent.NexusTile tile = NexusContent.NexusTile.newBuilder()
+                .setTile(NexusContent.TileData.newBuilder()
+                .setTileId(UUID.randomUUID().toString())
+                .setGridTile(
+                NexusContent.GridTile.newBuilder()
+                        .build())
+                .build())
+                .setSummary(NexusContent.TileSummary.newBuilder()
+                .setTileId("1")
+                .setBbox(NexusContent.TileSummary.BBox.newBuilder()
+                .setLatMin(51)
+                .setLatMax(55)
+                .setLonMin(22)
+                .setLonMax(30)
+                .build())
+                .setDatasetName("test")
+                .setDatasetUuid("4")
+                .setDataVarName("sst")
+                .setGranule("test.nc")
+                .setSectionSpec("0:1,0:1")
+                .setStats(NexusContent.TileSummary.DataStats.newBuilder()
+                .setCount(10)
+                .setMax(50)
+                .setMin(50)
+                .setMean(50)
+                .setMaxTime(500000)
+                .setMinTime(500000)
+                .build())
+                .addGlobalAttributes(NexusContent.Attribute.newBuilder()
+                .setName("day_of_year_i")
+                .addValues("006")
+                .build())
+                .addGlobalAttributes(NexusContent.Attribute.newBuilder()
+                .setName("attr_multi_value_test")
+                .addValues("006")
+                .addValues("multi")
+                .build())
+                .build())
+                .build()
+
+        def processingChainUnderTest = "$MODULE_NAME --$PROPERTY_NAME_SOLR_SERVER_URL=$SOLR_URL --$PROPERTY_NAME_SOLR_COLLECTION=$SOLR_CORE --$PROPERTY_NAME_S3_BUCKET=$S3_BUCKET_NAME --$PROPERTY_NAME_AWS_REGION=$AWS_REGION"
+
+        SingleNodeProcessingChainProducer chain = chainProducer(application, streamName, processingChainUnderTest)
+        SolrClient solrClient = singleNodeIntegrationTestSupport.getModule(streamName, MODULE_NAME, 0).applicationContext.getBean("solrClient", SolrClient)
+
+        chain.sendPayload(tile)
+
+        assertEqualsEventually 1, new Supplier<Integer>() {
+            @Override
+            Integer get() {
+                solrClient.commit()
+                def q = solrClient.query(new ModifiableSolrParams().add("q", "*:*"))
+                println "query: $q"
+                return q.results.numFound
+            }
+        }
+
+        assertEqualsEventually 1, new Supplier<Integer>() {
+            @Override
+            Integer get() {
+                ObjectListing objectListing = s3Client.listObjects(new ListObjectsRequest().withBucketName(S3_BUCKET_NAME))
+                return objectListing.getObjectSummaries().size()
+            }
+        }
+
+        chain.destroy()
+        //s3Mock.stop()
+
+    }
+
+    @Test(expected = ModuleConfigurationException.class)
```

Comment:
Move this to the bottom by the other tests that are like this one. Add more tests for the new validation rules mentioned above (If `cassandraKeyspace` set, require `cassandraContactPoints` etc...)